src/app/home/_component/mission_badge.tsx

```diff
@@ -0,0 +1,38 @@
+import { MoonIcon } from '@heroicons/react/20/solid';
+import { TicketIcon } from '@heroicons/react/20/solid';
+import { RocketLaunchIcon } from '@heroicons/react/20/solid';
+
+import Badge from '@/components/ui/badge';
+
+type Mission_type = 'DAILY' | 'WEEKLY' | 'SPECIAL';
+type Props = {
+  missionType: Mission_type;
+};
```

Comment:
작은 UI 컴포넌트를 만들 땐 props를 type으로 하였는데, 큰 덩어리의 컴포넌트들은 추후 확장할 일이 있을수도 있어서 interface로 선언하는게 어떨까요?

그리고 어차피 미션 뱃지 컴포넌트의 인자값이니 그냥 이름을 `type`으로 하여도 충분히 의미전달이 될 것 같습니다.

```suggestion
interface Props {
  type: MissionType;
};
```